refman/quickguide.tex

```diff
@@ -283,3 +283,15 @@ \subsubsection*{Substitution}
   thenHave(A, f(t) === t |- P(s) /\ s===t) 
       .by Substitution.ApplyRules(subst, f(t) === t)
 \end{lstlisting}
+
+
+\subsection*{File Options}
+Some options can be set at the start of a file, which will affect the behaviour of Lisa. These options are intended for use at a development stage.
+
+\vspace{2em}
+\begin{tabularx}{0.9\columnwidth}{|l|X|}
+  \lstinline|draft()| & Theorems outside of the curent file are assumed and not checked for correctness. 
```

Comment:
```suggestion
  \lstinline|draft()| & Theorems outside of the current file are assumed to be true and not checked for correctness. This can speed up repetitive runs during proof drafts.
```

Typo and a small addition.